### ISSUE TITLE
AbstractInput submitOnChange is a source of a11y failure

### ISSUE THREAD

**Issue by marksreeves** [BorderTech/wcomponents#1255]
We currently output **visible** a warning if `submitOnChange` is set. This is sufficient to prevent an _unexpected_ change of context and is usually enough to stop people using it. It is, however, time we dropped `submitOnChange` - it serves no useful purpose and should be replaced with a WAjaxControl and/or better design. 

This predates all of us I think and should have never been built.

See https://www.w3.org/TR/UNDERSTANDING-WCAG20/consistent-behavior-unpredictable-change.html
